dynet/expr.cc

```diff
@@ -26,7 +26,10 @@ Expression const_lookup(ComputationGraph& g, LookupParameter p, unsigned index)
 Expression const_lookup(ComputationGraph& g, LookupParameter p, const unsigned* pindex) { return Expression(&g, g.add_const_lookup(p, pindex)); }
 Expression const_lookup(ComputationGraph& g, LookupParameter p, const vector<unsigned>& indices) { return Expression(&g, g.add_const_lookup(p, indices)); }
 Expression const_lookup(ComputationGraph& g, LookupParameter p, const vector<unsigned>* pindices) { return Expression(&g, g.add_const_lookup(p, pindices)); }
-Expression zeroes(ComputationGraph& g, const Dim& d) { return Expression(&g, g.add_function<Zeroes>(d)); }
+Expression zeros(ComputationGraph& g, const Dim& d) { return Expression(&g, g.add_function<Constant>(d, 0.f)); }
+// Expression zeroes(ComputationGraph& g, const Dim& d) {return zeros(g, d);}
```

Comment:
It is actually here as an inline function:
https://github.com/clab/dynet/pull/769/files#diff-dca57b9f24bff4f4b8ab89e90a3057a8R389